Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -262,13 +262,14 @@ - (void)verifyChallengeAction:(nullable id)sender
 {
     [self startActivityIndicator];
     OWSProdInfo([OWSAnalyticsEvents registrationRegisteringCode]);
+    __weak CodeVerificationViewController *weakSelf = self;
```

Comment:
NBD. Using weak self will let this view be cleaned up a bit faster if user hits back during a long network request.